### ISSUE TITLE
daplunks dungeon level - giant puzzle

### ISSUE THREAD

**Issue by Sklore** [Sklore/RW_Community_Repository#2]
Have begun work on my level of the dungeon. 

This is based on a puzzle I saw at a local hedge maze. The concept being that you need to work from one room to the next but you can never cross the same colored corridor twice in a row. The goal is to work from the room at the bottom to the center of the map.

Once of the concerns I have is the difficulty level. From a player's perspective they will only see the rooms they have visited. My hope is that the players should map our the dungeon as they go physically as I feel it will be necessary for their success. It's not an easy task when you can see the whole thing let alone be restricted to the knowledge of just your room. 

I plan on adding some visual queues to warn of impending danger should they go the wrong way. Perhaps a dead body in the first left corridor indicating that blue > blue = death. Will likely do the same on the Red > Red at the very start. 

I'm also thinking alot about the level of punishment for going the wrong way. To make this work it needs to be a death experience. There are ways through the maze where someone could get to the middle by going the wrong way quite quickly if they soaked the damage. Need to think on this some as I don't like the idea of insta-kill traps. Perhaps insta-kill would be acceptable with enough visual warnings. Or perhaps the DM just needs to manage the damage to ensure they can't reach the middle without solving the puzzle first. 

Building on the ideas coming out of #1 perhaps this could be a final challenge to reach what ever the end of the dungeon happens to be. Or even a system for the very start designed to keep people out completely. 

![dungeon level_example](https://cloud.githubusercontent.com/assets/8046697/22227769/e10d7c3a-e21c-11e6-93c3-0333d28a6fdb.jpg)

**Comment by Sklore** [comment]
Ok one way I'm thinking the severity could be handled. 

Whenever a player steps into the corridor of a color that they had previously already passed a huge energetic force hits the player in the chest taking approx 25% of their health away and throwing them back into the room they came from. 

This teaches the players the dangers while ensuring they cannot simply run through and soak the damage but also sets the severity of the trap to be quite serious in that only a few attempts can be made before the players have a serious problem on their hands.

**Comment by Sklore** [comment]
Added in lighting effects.

![dungeon level2](https://cloud.githubusercontent.com/assets/8046697/22247508/4842ed4c-e28e-11e6-9dfb-be027316451d.JPG)

**Comment by Vargr-Ookami** [comment]
Looking good.

**Comment by Sklore** [comment]
I'm not sold on the lights yet. Prefer the original to be honest.

**Comment by Vargr-Ookami** [comment]
What if you tone down the intensitet of the light and / or make it more transparent?

**Comment by Sklore** [comment]
Yeah I'm thinking each light should have the same throw distance. Will run some tests asap. This would work fine as a DM version.

**Comment by Sklore** [comment]
Ok new version of the map which I'm much happier with. Have done both a Player and DM version. 

**Player Version**
![rw - community dungeon project - daplunk_player_s](https://cloud.githubusercontent.com/assets/8046697/22669571/1f377094-ed15-11e6-893a-085395276988.jpg)

**GM Version**
Contains spoilers. Only click if you want to see the solution. 
(https://drive.google.com/file/d/0B_6BXlzLTlj8MnhaVUc0WHVaMWM/view?usp=sharing)

**Comment by Sklore** [comment]
So entrance will be the lower portal. 
The room will be enough to keep people self-sufficient and alive for the rest of their life. They can chose to stay here or proceed forward. The bath will have healing properties. 

The arrows at the base of the statue are hints. As are the dead bodies. The concept is players can move from room to room only by crossing a corridor of a different color to the one they have previously passed. Crossing the same path twice will result in the player being shot with a force back into the room and taking significant damage. 

I'm playing around with the idea of making the treasure pile just north of the entrance into a trap. It will be a powerful sword (+3) with magical properties. But when picked up I'm considering activating all of the armored suits and turning them into defense systems designed to stop the sword from leaving. 

This could turn into a mini-game. Have the actual dungeon based on initiative turns where the players act, then the armor acts. Perhaps the armor will only move if it's within 2 rooms of the players? This adds an element of combat and excitement into the mix putting some pressure on the players. 

The well at the top will contain the same healing properties as the bath. 

I'm playing with the idea of a trap that makes people color blind. Perhaps the sword carrier loses the ability to see color?

The portal in the middle is the exit that will lead to the next dungeon.
